tests/integration/security/authn/permissive_test.go

```diff
@@ -101,22 +102,7 @@ spec:
 
 			var a echo.Instance
 			echoboot.NewBuilderOrFail(t, ctx).
-				With(&a, echo.Config{
-					Service:   "a",
-					Namespace: ns,
-					Galley:    g,
-					Pilot:     p,
-					Ports: []echo.Port{
-						{
-							Name:     "http",
-							Protocol: model.ProtocolHTTP,
-						},
-						{
-							Name:     "tcp",
-							Protocol: model.ProtocolTCP,
-						},
-					},
-				}).
+				With(&a, util.EchoConfig("a", ns, false, nil, g, p)).
```

Comment:
Will we need to pass non-false or non-nil argument to EchoConfig? It seems we won't, if so you can remove it from the argument list.

Reply:
At least one test is using it https://github.com/istio/istio/pull/14498/files#diff-142e7e2ec4553ec47315710067d9f4fbR55